plugins/TagFix_IntervalConditional.py

```diff
@@ -0,0 +1,77 @@
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyright Adrien Pavie 2019                                           ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+from plugins.Plugin import Plugin
+
+try:
+    from transporthours.main import Main
+    module_transporthours = True
+except ImportError as e:
+    print(e)
+    module_transporthours = False
```

Comment:
Simply import the module, an not almost silent fail.

Reply:
Done, I was using `TagFix_Opening_Hours.py` as a template, might not be a good one.

Reply:
It is the more buggy contrib of Osmose... issues still open and nobody able to fix it